blog/cross-account-eventbridge.md

```diff
@@ -0,0 +1,129 @@
+
+The flow of events for the order creation use case is as follows.
+
+1. An HTTP POST API is used to create an order. The backing Lambda function generates an order ID and sends an `Order.Created` event to the global bus.
+2. The delivery service picks up the `Order.Created` event from its local bus, processes the order , and sends a `Delivery.Updated` event including all the important delivery details to the global bus.
+3. The order service picks up the `Delivery.Updated` event from its local bus, and finally sends an `Order.Updated` event to the global bus.
+
+![Distributing cross-account events using a Global Bus](../local-global-eventbridge.png)
+
+> **Example Source**
+>  The full source code with documentation for this is available on [github.com/fourTheorem/cross-account-eventbridge](https://github.com/fourTheorem/cross-account-eventbridge/). It include a CDK pipeline for deployment of all resource to the three accounts.
+
+## Global bus event rules
+
+Events cannot be sent anywhere in EventBridge without a rule. Rules can be based on a schedule or an event pattern. For our backbone, we need to create pattern-based routing rules in the global bus. We create a single rule for each service account:
+
+```yaml
+eventPattern:
+  account:
+    - 'anything-but': 12345789012  # Anything but the source account
+```
+
+This rule will route all events to every service account except the one that sent the message.
+
+## Logging events for debugging and auditing
+
+One of the challenges people encounter with EventBridge when using it for the first time relates to observability. It can be difficult to understand which events are flowing through a bus and see their contents so that you can troubleshoot delivery failures. A simple way to address this is to create a rule to capture and log all events to CloudWatch Logs. How do you capture all events? EventBridge rules require you to have at least one condition in your filter, but a prefix match expression with an empty string will capture all events from any source:
+
+```yaml
+eventPattern:
+  source:
+    - prefix: ''
+```
+
+## Further reading and viewing
+
+We have covered the fundamental building blocks for a cross-account backbone with EventBridge. There is plenty more you can do with EventBridge, like using archives and event replaying, as well as integrating it into other AWS services. For a small amount of upfront effort and minimal ongoing maintenance, you can achieve a very flexible and scalable event bus for many applications across accounts.
+
+If you want to read more on EventBridge, [Luciano Mammino](https://twitter.com/loige) and I have written an article and have a YouTube video and podcast episode to accompany it:
+1. [fourTheorem Blog: What can you do with EventBridge]( https://www.fourtheorem.com/blog/what-can-you-do-with-eventbridge)
+2. [AWS Bites Episode 23: What’s the big deal with EventBridge? - YouTube](https://youtu.be/UjIE5qp-v8w)
```

Comment:
We could also link the article about SNS and mention that there is a comparison with EventBridge there as well.